src/admin/menu/views/MenuEdit.tsx

```diff
@@ -78,15 +67,13 @@ const MenuEdit: FunctionComponent<MenuEditProps> = ({ history, match }) => {
 				}
 			});
 		} else {
+			setPageType('create');
 			setIsLoading(false);
 		}
-	}, [match.params.id]);
+	}, [id]);
 
 	// Computed
-	const menuId = match.params.menu;
-	const pageTitle = `${startCase(menuId)}: item ${
-		pageType === 'create' ? 'toevoegen' : 'aanpassen'
-	}`;
+	const pageTitle = `${startCase(menu)}: item ${PAGE_TYPES_LANG[pageType]}`;
```

Comment:
de variable naam is wel wat moeilijker te begrijpen
wat is menu zonder context te hebben?
terwijl menuId duidelijk een string of int is

Kan je het mss menuKey of menuIdentifier noemen?

Reply:
Je kan die in de bovenste destructure idd hernoemen dan:
`const { menu: menuParent, id: MenuItemId } = match.params;`